src/firefly/queue_client.py

```diff
@@ -31,6 +37,9 @@ class QueueClient(QObject):
     # Signals responding to queue changes
     status_changed = Signal(dict)
     length_changed = Signal(int)
+    in_use_changed = Signal(bool)  # If length > 0, or queue is running
+    autostart_changed = Signal(bool)
```

Comment:
That's not quite true, though. *in_use* is true if the queue is started, or has items in it (i.e. it could be started). Right now I use this to determine whether it's useful to display the queue controls navbar.

I agree that this name is not clear. Any thoughts on what a better name would be?

Reply:
Thanks for the explanation, I think just as is is good~